From d3d5048e0a91f166687f58d537bc826f8a29d132 Mon Sep 17 00:00:00 2001
From: Alejandro Abdelnur <tucu@cloudera.com>
Date: Wed, 17 Apr 2013 15:27:59 -0700
Subject: [PATCH 053/596] MR1: CLOUDERA-BUILD. Add changes to mr1 build to
 specify JDK1.7 as target (cherry picked from commit
 098bc163405dfb3e5e61d4cf8e80fdfb47e21d24)

(cherry picked from commit 45a3883a507355916bf5e4e802b9a06b97e8c35f)
(cherry picked from commit 18379b27e6a698d45195af9db869b10708e90517)
(cherry picked from commit 216a244eee45cb14cb401cf37b935720291b158c)
---
 hadoop-mapreduce1-project/build.xml |   15 ++++++++-------
 1 file changed, 8 insertions(+), 7 deletions(-)

diff --git a/hadoop-mapreduce1-project/build.xml b/hadoop-mapreduce1-project/build.xml
index 8e8dc73..dacd6a7 100644
--- a/hadoop-mapreduce1-project/build.xml
+++ b/hadoop-mapreduce1-project/build.xml
@@ -375,6 +375,7 @@
 
   <target name="jvm-check">
     <property name="javaVersion" value="1.6"/>
+    <property name="targetJavaVersion" value="1.6"/>
 
     <property name="requiredJavaVersion" value="xx${javaVersion}"/>
     <property name="currentJavaVersion" value="xx${java.version}"/>
@@ -466,7 +467,7 @@
      destdir="${build.classes}"
      debug="${javac.debug}"
      optimize="${javac.optimize}"
-     target="${javac.version}"
+     target="${targetJavaVersion}"
      source="${javac.version}"
      deprecation="${javac.deprecation}">
       <compilerarg line="${javac.args} ${javac.args.warnings}" />
@@ -487,7 +488,7 @@
      destdir="${build.tools}"
      debug="${javac.debug}"
      optimize="${javac.optimize}"
-     target="${javac.version}"
+     target="${targetJavaVersion}"
      source="${javac.version}"
      deprecation="${javac.deprecation}">
       <compilerarg line="${javac.args} ${javac.args.warnings}" />
@@ -628,7 +629,7 @@
      destdir="${build.examples}"
      debug="${javac.debug}"
      optimize="${javac.optimize}"
-     target="${javac.version}"
+     target="${targetJavaVersion}"
      source="${javac.version}"
      deprecation="${javac.deprecation}">
       <compilerarg line="${javac.args} ${javac.args.warnings}" />
@@ -736,7 +737,7 @@
      destdir="${test.build.classes}"
      debug="${javac.debug}"
      optimize="${javac.optimize}"
-     target="${javac.version}"
+     target="${targetJavaVersion}"
      source="${javac.version}"
      deprecation="${javac.deprecation}">
       <compilerarg line="${javac.args} ${javac.args.warnings}" />
@@ -749,7 +750,7 @@
      destdir="${test.build.testjar}"
      debug="${javac.debug}"
      optimize="${javac.optimize}"
-     target="${javac.version}"
+     target="${targetJavaVersion}"
      source="${javac.version}"
      deprecation="${javac.deprecation}">
       <compilerarg line="${javac.args} ${javac.args.warnings}" />
@@ -766,7 +767,7 @@
      destdir="${test.build.testshell}"
      debug="${javac.debug}"
      optimize="${javac.optimize}"
-     target="${javac.version}"
+     target="${targetJavaVersion}"
      source="${javac.version}"
      deprecation="${javac.deprecation}">
       <compilerarg line="${javac.args} ${javac.args.warnings}"/>
@@ -1682,7 +1683,7 @@
         destdir="${build.anttasks}"
         debug="${javac.debug}"
         optimize="${javac.optimize}"
-        target="${javac.version}"
+        target="${targetJavaVersion}"
         source="${javac.version}"
         deprecation="${javac.deprecation}">
         <compilerarg line="${javac.args}"/>
-- 
1.7.9.5

